examples/vuejs-realworld-example-app/nginx.conf

```diff
@@ -0,0 +1,66 @@
+user nginx;
+worker_processes auto;
+
+events {
+    worker_connections 4096;
+}
+
+http {
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 120;
+    server_tokens off;
+
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    gzip on;
+    gzip_proxied any;
+    gzip_comp_level 6;
+    gzip_buffers 16 8k;
+    gzip_http_version 1.1;
+    gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
+
+    log_format combined_extra '$remote_addr - $remote_user [$time_local] '
+                              '"$request" $status $bytes_sent '
+                              '"$http_referer" "$http_user_agent" '
+                              '$request_time';
+
+    access_log /dev/stdout combined_extra;
+    error_log /dev/stderr warn;
+
+    server {
+        listen      80;
+        server_name localhost _;
+        root        /app/dist;
+
+        gzip on;
+        gzip_comp_level 6;
+        gzip_types *;
+
+        client_max_body_size 2M;
+
+        location = /_ping {
+            access_log off;
+            allow 127.0.0.1;
+            deny all;
+            return 200 "pong";
+        }
+
+        location /img {
+            alias /app/build/assets;
+
+            expires 1y;
+            access_log off;
+            add_header Cache-Control "public";
+        }
+
+        location / {
+            expires 60s;
```

Comment:
js and css assets are served via this location for vue js ?

Reply:
iirc `index.html` is in the root directory and assets are stored in `assets` or in `js/` and `css/`.

Reply:
should there have also a dedicated nginx location for such assets with an appropriate cache control header (as it is for the angularjs example) ?